SoundSwitch/Framework/NotificationManager/MMNotificationClient.cs

```diff
@@ -53,8 +53,11 @@ public void OnDefaultDeviceChanged(DataFlow flow, Role role, string defaultDevic
         {
             Task.Factory.StartNew(() =>
             {
-                var device = _enumerator.GetDevice(defaultDeviceId);
-                DefaultDeviceChanged?.Invoke(this, new DeviceDefaultChangedEvent(device, role));
+                if (defaultDeviceId != null)
```

Comment:
Invert the check and put it outside the Task.Factory.StartNew

I prefer to use early exit condition then nested if :)

Reply:
Done. :)